docs/source/modules/faker_sample.py

```diff
@@ -0,0 +1,43 @@
+"""
+Ideas to play with
+ - plotting max count or median max counts as function of samples?
+ - also keep track of unique name counts
+ - do similar for male and female names, or all at once?
+"""
+import sys
+import argparse
+
+from faker import Faker
```

Comment:
So I ended up using the `names` module instead of `faker`. https://github.com/treyhunner/names
It's not as popular, but I just couldn't stand the way faker names were duplicated so often.